core/common/src/test/java/alluxio/ConfigurationTest.java

```diff
@@ -11,319 +11,193 @@
-    Assert.assertEquals(Constants.SECOND_MS, intValue);
-
-    intValue = Configuration.getInt(Constants.WORKER_WORKER_BLOCK_THREADS_MIN);
-    Assert.assertEquals(256, intValue);
-
-    intValue = Configuration.getInt(Constants.WORKER_SESSION_TIMEOUT_MS);
-    Assert.assertEquals(10 * Constants.SECOND_MS, intValue);
-
-    intValue = Configuration.getInt(Constants.WORKER_NETWORK_NETTY_BOSS_THREADS);
-    Assert.assertEquals(1, intValue);
-
-    intValue = Configuration.getInt(Constants.WORKER_NETWORK_NETTY_WORKER_THREADS);
-    Assert.assertEquals(0, intValue);
+  public void getMalformedIntThrowsException() {
+    Configuration.set("key", "9448367483758473854738");
+    mThrown.expect(RuntimeException.class);
+    Configuration.getInt("key");
+  }
 
-    long longValue = Configuration.getBytes(Constants.WORKER_MEMORY_SIZE);
-    Assert.assertEquals(Constants.GB, longValue);
+  @Test
+  public void getLong() {
+    Configuration.set("key", "12345678910"); // bigger than MAX_INT
+    Assert.assertEquals(12345678910L, Configuration.getLong("key"));
   }
 
-  /**
-   * Tests the default properties for the user.
-   */
   @Test
-  public void userDefaultTest() {
-    int intValue = Configuration.getInt(Constants.USER_FAILED_SPACE_REQUEST_LIMITS);
-    Assert.assertEquals(3, intValue);
+  public void getMalformedLongThrowsException() {
+    Configuration.set("key", "999999999999999999999999999999999999"); // bigger than MAX_LONG
+    mThrown.expect(RuntimeException.class);
+    Configuration.getLong("key");
+  }
 
-    intValue = Configuration.getInt(Constants.USER_HEARTBEAT_INTERVAL_MS);
-    Assert.assertEquals(Constants.SECOND_MS, intValue);
+  @Test
+  public void getDouble() {
+    Configuration.set("key", "1.1");
+    Assert.assertEquals(1.1, Configuration.getDouble("key"), /*tolerance=*/0.0001);
+  }
 
-    long longValue = Configuration.getBytes(Constants.USER_FILE_BUFFER_BYTES);
-    Assert.assertEquals(Constants.MB, longValue);
+  @Test
+  public void getMalformedDoubleThrowsException() {
+    Configuration.set("key", "1a");
+    mThrown.expect(RuntimeException.class);
+    Configuration.getDouble("key");
+  }
 
-    longValue = Configuration.getBytes(Constants.USER_BLOCK_REMOTE_READ_BUFFER_SIZE_BYTES);
-    Assert.assertEquals(8 * Constants.MB, longValue);
+  @Test
+  public void getFloat() {
+    Configuration.set("key", "1.1");
+    Assert.assertEquals(1.1, Configuration.getFloat("key"), /*tolerance=*/0.0001);
   }
 
-  /**
-   * Tests the simple substitution of variables.
-   */
   @Test
-  public void variableSubstitutionSimpleTest() {
-    Configuration.merge(sTestProperties);
+  public void getMalformedFloatThrowsException() {
+    Configuration.set("key", "1a");
+    mThrown.expect(RuntimeException.class);
+    Configuration.getFloat("key");
+  }
 
-    String home = Configuration.get("home");
-    Assert.assertEquals("hometest", home);
+  @Test
+  public void getBoolean() {
+    Configuration.set("key", "true");
```

Comment:
also test `Configuration.set("key", "false");`
and also in different cases like uppercases?